config.example.yaml

```diff
@@ -0,0 +1,138 @@
+networks:
+  - id: "localosmosis"
```

Comment:
Would be nice to comment here that I think the ID is the thing that points to the context json file?